ee/packages/license/src/index.ts

```diff
@@ -2,3 +2,4 @@ export { DuplicatedLicenseError } from './errors/DuplicatedLicenseError';
 export * from './licenseImp';
 export * from './MockedLicenseBuilder';
 export * from './applyLicense';
+export * from './AirGappedRestriction';
```

Comment:
from what I could see there is no "init" action that needs to be done, so there is not need to import the class/instance here.
```suggestion
```